WSL/about.md

```diff
@@ -20,12 +18,32 @@ You can:
 * Run common command-line tools such as `grep`, `sed`, `awk`, or other ELF-64 binaries.
 * Run Bash shell scripts and GNU/Linux command-line applications including:  
     *Tools: vim, emacs, tmux
-    *Languages: Javascript, Ruby, Python, C/C++, C# & F#, Rust, Go, etc.
-    *Services: SSHD, MySQL, Apache, lighttpd
+    *Languages: [NodeJS](https://docs.microsoft.com/windows/nodejs/setup-on-wsl2), Javascript, [Python](https://docs.microsoft.com/windows/python/web-frameworks), Ruby, C/C++, C# & F#, Rust, Go, etc.
+    *Services: SSHD, MySQL, Apache, lighttpd, [MongoDB](https://docs.microsoft.com/windows/nodejs/databases), [PostgreSQL](https://docs.microsoft.com/windows/python/databases).
 * Install additional software using own GNU/Linux distribution package manager.
 * Invoke Windows applications using a Unix-like command-line shell.
 * Invoke GNU/Linux applications on Windows.
 
-## Windows Subsystem for Linux 2
+## What is WSL 2?
 
-WSL 2 is a new version of the architecture that powers the Windows Subsystem for Linux to run ELF64 Linux binaries on Windows. Its primary goals are to increase file system performance, as well as adding full system call compatibility. This new architecture changes how these Linux binaries interact with Windows and your computer's hardware, but still provides the same user experience as in WSL 1 (the current widely available version). Individual Linux distros can be run either as a WSL 1 distro, or as a WSL 2 distro, can be upgraded or downgraded at any time, and you can run WSL 1 and WSL 2 distros side by side. WSL 2 uses an entirely new architecture that uses a real Linux kernel.
+WSL 2 is a new version of the Windows Subsystem for Linux architecture that powers the Windows Subsystem for Linux to run ELF64 Linux binaries on Windows. Its primary goals are to **increase file system performance**, as well as adding **full system call compatibility**.
+
+This new architecture changes how these Linux binaries interact with Windows and your computer's hardware, but still provides the same user experience as in WSL 1 (the current widely available version).
+
+Individual Linux distributions can be run with either the WSL 1 or WSL 2 architecture. Each distribution can be upgraded or downgraded at any time and you can run WSL 1 and WSL 2 distributions side by side. WSL 2 uses an entirely new architecture that benefits from running a real Linux kernel.
+
+## Next steps
+
+    - [Install WSL 1 and update to WSL 2](./install-win10.md)
+
+    - [Comparing WSL 2 and WSL 1](./compare-versions.md)
+    
+    - [ ]()
```

Comment:
We can't merge these into master without this table being filled out or removed.